src/topology/algebra/ordered.lean

```diff
@@ -2536,6 +2536,11 @@ lemma intermediate_value_Icc' {a b : α} (hab : a ≤ b) {f : α → δ} (hf : c
   Icc (f b) (f a) ⊆ f '' (Icc a b) :=
 is_preconnected_Icc.intermediate_value (right_mem_Icc.2 hab) (left_mem_Icc.2 hab) hf
 
+/--Intermediate Value Theorem for continuous functions on closed intervals, unordered case.-/
+lemma intermediate_value_interval {a b : α} {f : α → δ} (hf : continuous_on f (interval a b)) :
+  interval (f a) (f b) ⊆ f '' interval a b :=
+by cases le_total (f a) (f b); simp [h, is_preconnected_interval.intermediate_value, hf]
```

Comment:
Use `cases ... with h`, since you use the auto-generated name.

Reply:
Thanks, I ought to have done this - bad form on my part. But I will actually alter the proof to avoid using the author-genreated name by using `*` instead of the hypotheses names.